packages/story-editor/src/app/config/useConfig.ts

```diff
@@ -18,12 +18,17 @@
  * External dependencies
  */
 import { identity, useContextSelector } from '@googleforcreators/react';
+
 /**
  * Internal dependencies
  */
+import type { ConfigState } from '../../types/configProvider';
 import Context from './context';
 
-function useConfig(selector) {
+function useConfig(): ConfigState;
```

Comment:
This is actually only used like this, it seems (without using props).

Reply:
Makes sense, because the state never changes anyway.